storage/tools/dump_tree/dumplib/nochange_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dumplib
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+// TestDBFormatNoChange ensures that the prefix, suffix, and protos stored in the database do not change.
+// This test compares the output from dump_tree against a previously saved output.
+func TestDBFormatNoChange(t *testing.T) {
+	savedFile := "../../../../testdata/dump_tree_output"
+	out := Main(871, 50, "Leaf %d", true, false, false, false, false, true, false, false)
```

Comment:
Can you add a couple more tests on different tree sizes to this. Should include a perfect tree size e.g. 1024 and a non perfect tree with an even number of leaves e.g. 1000 maybe. Also one that should fit into one subtree e.g. 96.